warehouse/rss/views.py

```diff
@@ -10,13 +10,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from datetime import timedelta
+
+from pyramid.httpexceptions import HTTPBadRequest
 from pyramid.view import view_config
 from sqlalchemy.orm import joinedload
 
 from warehouse.cache.origin import origin_cache
 from warehouse.packaging.models import Project, Release
+from warehouse.utils import now
 from warehouse.xml import XML_CSP
 
+DEFAULT_RESULTS = 40
+MAX_RESULTS = 200
+
+
+def _get_int_query_param(request, param, default=None):
```

Comment:
Nit: I feel defaulting to `None` might be dangerous here as the caller might expect an integer. The two options I see is raising an expection or returning a sentinel value (maybe `-1` or `0` would make sense?).